backend/models/userTasks.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require("mongoose");
+
+const taskSchema = mongoose.Schema(
+    {
+        title: {
+            type: String,
+            required: true
+        },
+        user_id: {
+            type: mongoose.Schema.Types.ObjectId
```

Comment:
I believe the `user_id` should be a required value too no? (Since we'll need to always identify a task with the user that created it)

Reply:
Fixed: Changes are reflected in recent commits